pkg/kn/commands/revision/human_readable_flags.go

```diff
@@ -52,10 +52,10 @@ func printRevisionList(revisionList *servingv1alpha1.RevisionList, options hprin
 	return rows, nil
 }
 
-// printRevision populates the knative revision table rows
+// printRevision populates the Knative revision table rows
 func printRevision(revision *servingv1alpha1.Revision, options hprinters.PrintOptions) ([]metav1beta1.TableRow, error) {
-	service := revision.Labels[serving.ConfigurationLabelKey]
 	name := revision.Name
+	service := revision.Labels[serving.ConfigurationLabelKey]
```

Comment:
I'm still not sold that it should be the `ConfigurationLabelKey`. It's just by accident that the Configuration is named like the service. You should use `ServiceLabelKey` instead.

Reply:
```
  // ServiceLabelKey is the label key attached to a Route and Configuration indicating by
  // which Service they are created.
  ServiceLabelKey = GroupName + "/service"
```
@rhuss : yup, updating..